### ISSUE TITLE
[Help request] Interpretation of KShape-clustering output

### ISSUE THREAD

**Issue by toshaklg** [tslearn-team/tslearn#362]
Hello there!

I am using tslearn for clustering some some [NDVI](https://en.wikipedia.org/wiki/Normalized_difference_vegetation_index) data I've got and I have a problem with results interpretation after KShape clustering. 

In your [docs](https://tslearn.readthedocs.io/en/stable/auto_examples/clustering/plot_kshape.html?highlight=kshape#kshape) there is a quite nice example that I followed.
Basically, the NDVI data I have is **far** from ideal, but it is in time-series format, each entry has the same length and values lay between 0 and 1. Under such conditions, KMeans with DTW works somewhat decent and results resemble what I have:

![vcxsrv_nRWJNoK0YK](https://user-images.githubusercontent.com/43007372/133795533-a9c000ba-b63a-44e0-815e-9164c4425915.png)

However, in case with KShape feels like the data is messed up after applying
```
ndvi_input_scaled = TimeSeriesScalerMeanVariance().fit_transform(ndvi_input)
```

and instead of nice lines that I get in KMeans, and you have in the [example](https://tslearn.readthedocs.io/en/stable/auto_examples/clustering/plot_kshape.html?highlight=kshape#kshape), I am getting some bizzare -2.5 values that jump like crazy over the period and they do not look even remotely similar to what I have as an input.

![vcxsrv_l8qHCm0Cdx](https://user-images.githubusercontent.com/43007372/133795307-8ee7d4fb-4e27-4697-8de5-902ca039c7e0.png)

So, am I missing something or the data is bad? Any suggestions, maybe?

Thanks!

**Comment by kushalkolar** [comment]
`TimeSeriesScalerMeanVariance().fit_transform()` will transform your data into standard deviation units, that's what you're seeing
